bad-bits/lib/store.js

```diff
@@ -7,38 +7,42 @@
  * @param {string} etag - ETag for the current denylist
  */
 export async function updateBadBitsDatabase(env, currentHashes, etag) {
-  try {
```

Comment:
I moved the try/catch block to `fetchAndStoreBadBits` as it makes the code in `updateBadBitsDatabase` easier to read, IMO.